lib/audited/audit.rb

```diff
@@ -26,16 +26,16 @@ class Audit < ::ActiveRecord::Base
 
     serialize :audited_changes
 
-    scope :ascending,     ->{ reorder(version: :asc) }
-    scope :descending,    ->{ reorder(version: :desc)}
-    scope :creates,       ->{ where(action: 'create')}
-    scope :updates,       ->{ where(action: 'update')}
-    scope :destroys,      ->{ where(action: 'destroy')}
-
-    scope :up_until,      ->(date_or_time){where("created_at <= ?", date_or_time) }
-    scope :from_version,  ->(version){where(['version >= ?', version]) }
-    scope :to_version,    ->(version){where(['version <= ?', version]) }
-    scope :auditable_finder, ->(auditable_id, auditable_type){where(auditable_id: auditable_id, auditable_type: auditable_type)}
+    scope :ascending,     -> { reorder(version: :asc) }
+    scope :descending,    -> { reorder(version: :desc) }
+    scope :creates,       -> { where(action: 'create') }
+    scope :updates,       -> { where(action: 'update') }
+    scope :destroys,      -> { where(action: 'destroy') }
+
+    scope :up_until,      ->(date_or_time) { where('created_at <= ?', date_or_time) }
```

Comment:
If you're going to make the spacing changes after the arrow on lines 29-33, you should do the same on the scopes from 35-38.